### DISCUSSION TITLE
UPDATE: FIXED Jekyll site builds locally but looks odd with no errors on Github pages

### DISCUSSION THREAD

**Discussion by maw501** [community/community#22608] [Other Feature Feedback, Questions, & Ideas]
<p>Hello,</p>
<p>I am trying to build a site using Jekyll and Github pages. Everything seems okay locally but when I push to git and navigate to it the website looks a mess. Unfortunately I don’t seem to be getting any build errors in order to diagnose.</p>
<p>The github <a href="https://github.com/maw501/maw501.github.io" rel="nofollow noopener">repo</a> and website <a href="https://maw501.github.io/" rel="nofollow noopener">here</a>.</p>
<p>I’m a html newbie - I built the site by forking a <a href="http://themes.jekyllrc.org/herringcove-theme/" rel="nofollow noopener">theme</a> (herring cove) and then hacking about to get things to look as I wished. I’m hoping it’s something simple but am at a bit of a loss at the moment with no errors to go on.</p>
<p>Many thanks in advance,</p>
<p>Mark</p>
<p>P.S. If you don’t have time to reply but know what to google in order for me to debug please let me know (as in, I don’t even know what to call the ‘error’ when searching).</p>

**Comment by nadiajoyce** [comment]
<p>Hi <a class="mention" href="/maw501">@maw501</a>,</p>
<p>I’m so glad you were able to find a solution to your question! If you could, would you please comment here letting us all know how you fixed the problem? The information could be really helpful to other GitHub users if they run into the same question. Thanks!</p>

**Comment by maw501** [comment, marked as answer]
<p>Hi <a class="mention" href="/nadiajoyce">@nadiajoyce</a>,</p>
<p>I am not fully sure what the issue was though I think it related to having a different set-up locally to what is used by github-pages. I followed instructions <a href="https://help.github.com/articles/setting-up-your-github-pages-site-locally-with-jekyll/" rel="nofollow noopener">here</a> to update my local environment to the correct gem (which itself threw errors I can’t recall - though I was able to google my way out of them).</p>
<p>Thanks</p>

**Comment by nadiajoyce** [comment]
<p>Thanks so much, <a class="mention" href="/maw501">@maw501</a>! I’m sure this will help others :slight_smile:</p>
